packages/terra-application-utility/src/utility/_UtilityMenuItem.jsx

```diff
@@ -127,6 +132,7 @@ class UtilityMenuItem extends React.Component {
     const bodyItemClassNames = cx([
       { 'header-utility-body-item': variant === Utils.VARIANTS.HEADER },
       { 'menu-utility-body-item': variant === Utils.VARIANTS.MENU },
+      { 'read-only': isReadOnly },
```

Comment:
If this is readOnly, would we need to change the role type of the list item and remove the click / keydown handlers?

Reply:
Thinking something like this to address this:
```
const roles = isReadOnly ? 'note' : 'button';
/* eslint-disable jsx-a11y/no-static-element-interactions, jsx-a11y/no-noninteractive-element-to-interactive-role */
const renderBodyItem = (fill, wrapOnKeyDown, handleSelection) => (
  <li
    {...customProps}
    tabIndex={!isReadOnly ? '0' : undefined}
    key={itemKey}
    onClick={!isReadOnly ? event => handleSelection(event, itemKey) : undefined}
    onKeyDown={!isReadOnly ? wrapOnKeyDown(itemKey, onKeyDown) : undefined}
    role={roles}
    className={bodyItemClassNames}
    aria-label={title}
    ref={this.setItemNode}
  >
    <Arrange
      fitStart={leftInset ? <IconCheckmark className={checkmarkClassNames} /> : null}
      fill={fill}
      fillAttributes={{ className: cx('menu-item-fill') }}
      fitEnd={rightInset ? <IconChevronRight className={chevronClassNames} /> : null}
      align="center"
      className={arrangeClassNames}
    />
  </li>);
```
Lint pops up an error about the tabindex on a non-interactive element, but if I do what is in the code above, that creates an accessibility issue (screen readers won't be able to tab to it to read it). Otherwise, I can just disable the rule for that line.

Reply:
What you have looks good to me. role="note" seems fine to me as well, hadn't heard of it before so I had to look it up.